src/pepr/operator/controllers/network/generators/kubeAPI.ts

```diff
@@ -1,9 +1,13 @@
 import { V1NetworkPolicyPeer } from "@kubernetes/client-node";
-import { K8s, kind, Log, R } from "pepr";
+import { K8s, kind, R } from "pepr";
 
+import { setupLogger } from "../../../../logger";
 import { RemoteGenerated } from "../../../crd";
 import { anywhere } from "./anywhere";
 
+// configure subproject logger
+const log = setupLogger("operator.generators");
+
```

Comment:
Should this just be `operator.network`? Or does that violate the structure we were trying to set?

Reply:
yeah based on the initial structure it would be different since we went with top most directory followed by bottom most directory. Kind of why initially i went with something like `operator.network.generators` but does start to get long.